quarkus/service/src/intTest/java/org/apache/polaris/service/quarkus/it/relational/jdbc/JdbcQuarkusRestCatalogViewGcpIT.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.quarkus.it.relational.jdbc;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.TestProfile;
+import org.apache.polaris.service.it.test.PolarisRestCatalogViewGcpIntegrationTest;
+
+@TestProfile(RelationalJdbcProfile.class)
+@QuarkusIntegrationTest
+public class JdbcQuarkusRestCatalogViewGcpIT extends PolarisRestCatalogViewGcpIntegrationTest {}
```

Comment:
Do we actually need per-cloud tests? Is there something specific to a cloud vendor about the JDBC metastore?

Reply:
Honestly no, i think though we do need to override creds which in my understanding is cloud provider specific, but i don;t think that happening infact all CP tests are like no-op, so not sure why and what was the background for that ?

Reply:
I'd think as far as MetaStore testing going, using FILE storage should be sufficient.